jovian/jovian_nb_ext/main.js

```diff
@@ -741,19 +735,27 @@ define([
     jvn_notif.element.attr("disabled", true);
   }
 
-  function dropdownOption(modal, option) {
+  function openModal(func, modal = null) {
+    // Helper function; which use to open a new window(modal)
+    // from an existing window(modal/dialog)
     return new Promise(res => {
-      modal.modal("hide");
+      let len = $(".modal-backdrop").length;
+      if (modal != null) {
+        modal.modal("hide");
+      }
```

Comment:
There is no instance where you have used the `modal` parameter. You can remove it if its not being used